indico/modules/rb_new/controllers/backend/rooms.py

```diff
@@ -106,3 +106,20 @@ def _process_PUT(self):
     def _process_DELETE(self):
         session.user.favorite_rooms.discard(self.room)
         return '', 204
+
+
+# TODO
+class RHRoomUpdate(RHRoomBase):
+    # TODO: Check if current user is allowed to edit room
+    def _check_access(self):
+        RHRoomBase._check_access(self)
+        if session.user.is_admin is False:
```

Comment:
- don't use `is False`
- we have `rb_is_admin(session.user)` to check if someone is a room booking admin